front/lib/connector_providers.ts

```diff
@@ -153,12 +152,19 @@ export function getDataSourceNameFromView(dsv: DataSourceViewType): string {
   return dsv.dataSource.name;
 }
 
-type LogoType = ((props: SVGProps<SVGSVGElement>) => JSX.Element) | LucideIcon;
+export function getConnectorProviderLogo(
```

Comment:
This one was removed now all calls should go through `getConnectorProviderLogoWithFallback`